test/rules/no-object-literal-type-assertion/default/test.ts.lint

```diff
@@ -16,6 +16,10 @@ x as T;
 {} as unknown;
 <unknown> {};
 
+// Allow const assertion
+({}) as const;
+<const> ({});
```

Comment:
See https://github.com/palantir/tslint/blob/c01bb2cdc258da4f9727781151d91773fba3e283/test/rules/no-unused-variable/default/var.ts.lint for an example of how to only include some lines of tests in particular TypeScript versions. That should help with your CI failures!

Reply:
Thanks!